docsite/rst/committer_guidelines.rst

```diff
@@ -0,0 +1,72 @@
+
+Roles on Core
+=============
+* Core Committers: Fine to do PRs for most things, but we should have a timebox. Hanging PRs may merge on the judgement of these devs.
+* Module Owners: Module Owners own specific modules and have indirect commit access via the current module PR mechanisms.
+
+General Rules
+=============
+Individuals with direct commit access to ansible/ansible (+core, + extras) are entrusted with powers that allow them to do a broad variety of things--probably more than we can write down. Rather than a list of what you *can* do, this is a list of what you *should not* do and, in lieu of anything else, individuals with this power are expected to use their best judgement. 
+
+* Don’t commit directly.
+* PRs that have tests will be looked at with more priority than PRs without tests. Of course not all changes require tests, but for bug fixes or functionality changes, please add tests.
+* Documentation. If your PR is new feature or a change to behavior, make sure you’ve updated associated documentation or notified the right people to do so. It also helps to add the version of Core against which this documentation is compatible (to avoid confusion with stable versus devel docs, for backwards compatibility, etc.).
+* Someone else should merge your pull requests. If you are a Core Committer you have leeway here for minor changes.
+* After a merge clean up dead forks/branches. Don’t leave a mess hanging around.
+* Consider backwards compatibility (don’t break existing playbooks).
+* Consider alternate environments (yes, people have bad environments, but they are the ones that need us the most).
+* Always discuss the technical merits, never address the person’s limitations (you can later go for beers and call them idiots, but not in IRC/Github/etc).
+* Consider the maintenance burden, some things are cool to have, but might not be worth shoehorning in.
+* Complexity breeds all kinds of problems, so keep it simple.
+* Lastly, comitters that have no activity on the project (merges, triage, commits, etc) will have permissions suspended.
+
+Committers are expected to continue to follow the same community and contribution guidelines followed by the rest of the Ansible community. 
+
+
+People
+======
+Individuals who have been asked to become part of this group have generally been contributing in significant ways to the Ansible community for some time. Should they agree, they are requested to add their names & github IDs to this file below via pull request, indicating that they agree to act in the ways that their fellow committers trust that they will act. 
```

Comment:
editing suggestion:  Individuals who've been asked to become a part of this group have generally been contributing in significant ways to the Ansible community for some time. Should they agree, they are requested to add their names and GitHub IDs to this file, in the section below, via a pull request. Doing so indicates that these individuals agree to act in the ways that their fellow committers trust that they will act.